Makefile.toml

```diff
@@ -38,9 +39,17 @@ if ${is_all_in_one_enabled}
 else
   set_env RISEDEV_CARGO_BUILD_CRATE "risingwave_cmd"
 end
+
+if ${is_hdfs_backend}
+  set_env BUILD_HDFS_BACKEND_CMD "-p risingwave_object_store --features hdfs-backend"
```

Comment:
why do we need to add `-p risingwave_object_store`? I think adding `--features hdfs-backend` is good enough.

Reply:
It seems that `RISEDEV_CARGO_BUILD_CRATE` do not include `risingwave_object_store`. I will double check this and modify if necessary.

Reply:
We need to add `-p risingwave_object_store`:
<img width="935" alt="image" src="https://user-images.githubusercontent.com/58715567/218074015-2c0f0ae2-d86c-4841-9db8-47252fa882f6.png">

Reply:
It includes `risingwave_cmd` or `risingwave_cmd_all`, which is basically the risingwave binary so `risingwave_object_store` should be transitively included.

Reply:
`risingwave_cmd` do not have direct dependency of `rw_object_store`,, so let's keep it simple and do not add more dependency here.